src/components/Sidebar/Sidebar.jsx

```diff
@@ -70,11 +68,6 @@ const StyledAside = styled.aside`
     border-width: 0;
   }
 
-  &.opened,
```

Comment:
We can't close the Sidebar anymore.

Reply:
This is weird...
By the way, why did you apply `appear` on `Sidebar` but not on `Toolbar` ?

Reply:
Sidebars are meant to be shown and hidden. They are not part of the "hard" UI such as the main content and the Toolbars which are fixed (the Toolbar collapse mode is an exception and there is always a fixed button).

For me, Sidebar, Modal and Notification are ephemeral components.

Reply:
Okay.
One thing though.
The `_triggerCallback()` function has no real use with the `onExited` prop of `Transition`. Is it okay if I remove it ?

Reply:
I'm not sure to understand your point. So I kind of agree with you if you want to clean the code ;)

I just want to be sure that the call back props will still be present and working though. Is it the case?

Thanks!

Reply:
Do not merge the PR now, I will submit a modification for this, and we'll discuss it before merging.

Reply:
Okay, I must apologize.

> I need to add that I'm not sure what your intention with the Toolbar is.

There is typo, I meant "with the Sidebar", which is mounted and dismounted (unlike the `Toolbar`).

The chunk of code I quoted:
```js
if (this.props.opened !== nextProps.opened) {
      if (nextProps.opened === true) {
        this._triggerCallback('onOpen');
      } else {
        this._triggerCallback('onClose');
      }
}
```
was taken from the `Sidebar` component, as all the comments I made on the subject.

I cleaned the `Sidebar` code to get rid of useless `_triggerCallback()`.
This is done, I have cleaned the code, check if the corresponding code fits your expectations.

Reply:
Thank you!